behat.yml.dist

```diff
@@ -77,6 +77,13 @@ default:
                 - DataFixturesContext
                 - ApiContext
 
+        api-projects:
```

Comment:
Also, add this new suite to `.github/workflows/tests_behat.yml` to run this suite via GitHub actions